.github/CODEOWNERS

```diff
@@ -18,7 +18,7 @@
 #/avm/res/authorization/role-definition/ @Azure/avm-res-authorization-roledefinition-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/automation/automation-account/ @Azure/avm-res-automation-automationaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/batch/batch-account/ @Azure/avm-res-batch-batchaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/cache/redis/ @Azure/avm-res-cache-redis-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/cache/redis/ @Azure/avm-res-cache-redis-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Team must be created

Reply:
@Azure/avm-res-cache-redis-module-owners-bicep
@Azure/avm-res-cache-redis-module-contributors-bicep

Reply:
It still shows errors, as the link to the parent team may not be established/approved yet

Reply:
both teams exist and parents are assigned